xarray/tests/test_utils.py

```diff
@@ -23,11 +23,13 @@ def new_method():
 
 
 @pytest.mark.parametrize(
-    "a, b, expected", [["a", "b", np.array(["a", "b"])], [1, 2, pd.Index([1, 2])]]
+    ["a", "b", "expected"],
+    [
+        [np.array(["a"]), np.array(["b"]), np.array(["a", "b"])],
+        [np.array([1], dtype="int64"), np.array([2], dtype="int64"), pd.Index([1, 2])],
```

Comment:
here, the idea is to avoid the different default precision on windows by explicitly setting the `int` precision on construction